### ISSUE TITLE
[Quest] [Mage] Investigate the Alchemist Shop

### ISSUE THREAD

**Issue by Zelfie** [SunwellWoW/Sunwell-TBC-Bugtracker#2398]
Decription:
Same as the Alliance "Investigate the Blue Recluse" quest, this quest is currently broken.

How it works:
After lowering voids hp till 10% or killing him, he becomes untargetable, but is stunned. We cant use the Chest of Containment Coffers on him. There also spawns a 2nd void, which is also untargetable, but attacks us. These ghosts also have a huge range, and can follow you all the way around Undercity.

How it should work:
You should be able to trap the ghost, 

Source (you should point out proofs of your report, please give us some source): 

https://github.com/SunwellWoW/Sunwell-TBC-Bugtracker/issues/1717
Alliance version.

**Comment by brimosimo** [comment]
Yes it's bugged. Try pulling them over to the guards. You probably die. Resurrect, aggro a single one and pull it back to trap it as soon as it drops below 10%. Kind of a hassle but doable.
